script/1_0_2current_en_script/Ciel/Day 2/02_01_CIEL02_1_2.txt

```diff
@@ -15,7 +15,7 @@ It was more crowded than usual because of the new products they brought in, so i
 And so, I've got some leftover side dishes... Since you're about to eat, would you like a little side menu with your lunch?"
 <Page6>
 "――――――"
-I observe a group of girls to make sure they don't notice me.
+I observe a group of girls making sure they don't notice me.//気付かれないよう女子グループを盗み見る。
```

Comment:
I would add a comma after girls, otherwise it reads like he is observing the girls while the girls actively work to avoid noticing Shiki